waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/CloseableThriftHiveMetastoreIfaceClientFactory.java

```diff
@@ -59,6 +63,7 @@ public CloseableThriftHiveMetastoreIface newInstance(AbstractMetaStore metaStore
     }
     Map<String, String> properties = new HashMap<>();
     properties.put(ConfVars.METASTOREURIS.varname, uris);
+    properties.putAll(waggleDanceConfiguration.getConfigurationProperties());
```

Comment:
There should be a null check here, so this should only be done if `waggleDanceConfiguration.getConfigurationProperties ()` is not null.